acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/utils/NakadiTestUtils.java

```diff
@@ -111,7 +111,9 @@ public static void processEvents(final String path, final int count, final IntFu
         given()
                 .body("[" + events + "]")
                 .contentType(JSON)
-                .post(path);
+                .post(path)
+                .then()
+                .statusCode(HttpStatus.SC_OK);
```

Comment:
Not ignoring the status code here helps to spot wrong expectations in the tests instead of trying to figure out why we are consuming less events that we have (presumably) published.